### PR TITLE
[Sample][Accessibility] Reply button icon update

### PR DESCRIPTION
**Pull request by golddove**
## Related Issue
ADO 24043965

## Description
For 'reply' selectAction, use a representative icon instead of cat image.

Sourced from Microsoft Brand Central's Microsoft 365 illustrations.

## How Verified
Manual verification

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4774)

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
Looks good to me; Looks like the following additional changes requested by the Accessibility SMEs:
for different reply button please add screen reader name like below for each reply button:
1) Text input Reply button
2) Text input no icon Reply button
3) Multiline text input reply button.

**Review by shalinijoshi19 — CHANGES_REQUESTED** [review]
Additional changes/specifics requested by the E+D folks.

File: samples/v1.2/Tests/Input.Text.InlineAction.Multiline.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.2",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Text input with an inline action"
+		},
+		{
+			"type": "Input.Text",
```
Comment (almedina-ms) [inline samples/v1.2/Tests/Input.Text.InlineAction.Multiline.json:11]: you repeated the single line input elements here, you should remove them

**Review by almedina-ms — CHANGES_REQUESTED** [review]
Please update multiline sample

**Review by shalinijoshi19 — APPROVED** [review]

**Comment by shalinijoshi19** [comment]
Signing off at my end pursuant to addressing comments by others/ @almedina-ms ..Tx!

**Review by almedina-ms — APPROVED** [review]

**Pull request merged**